### DISCUSSION TITLE
Add prompt when clicking Approve or Save

### DISCUSSION THREAD

**Discussion by entrepeneur4lyf** [cline/cline#615] [Feature Requests]
I would like to be able to send the model a comment or directive when I click Save or Approve. Currently, anything I add to the chat is ignored.

**Comment by whitebearded** [comment]
+1 I miss this on a daily base too, big plus one :-) Esp. to prevent optimization loops when you're not really sure if Claude is really doing something substantial or not, and you save and save and save without wanting to interrupt it
